static/templates/actions_popover_content.hbs

```diff
@@ -52,6 +52,14 @@
         </a>
     </li>
     {{/if}}
+    {{#if should_display_hide_option}}
+    <li>
+        <a href="#" class="rehide_muted_user_message" data-message-id="{{message_id}}">
+            <i class="fa fa-minus" aria-hidden="true"></i>
+            {{t "Hide this message" }}
+        </a>
```

Comment:
Would it be more clear that this is related to user muting (and not a completely different feature, like "Collapse") if it included the word "again"? Something like `Hide this message again` or `Undo reveal`?

Reply:
Both of them look good to me. May be I'll change it to `Undo reveal` as it is short and not causing confusion between the adjacent terms in the list like `Hide` and `Collapse`. 
I'll update if I get further suggestion regarding this.

Reply:
How about this?

1. Put "Hide this message again" as the top item in the menu to make it easy to find.
2. Keep the rest of the menu the same as now for consistency.
3. Change the icon for "Hide this message again" to be an eye-slash to differentiate it from the icon for `Collapse`.

What do you think?

Reply:
Okk, I changed it as @alya suggested.